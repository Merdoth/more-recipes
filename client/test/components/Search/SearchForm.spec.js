import React from 'react';
import { shallow } from 'enzyme';
import expect from 'expect';
import { SearchForm } from
  '../../../components/Search/SearchForm.jsx';


const setup = () => shallow(<SearchForm />);

describe('SearchForm Component snapshot', () => {
  it('it should render the right amount of elements', () => {
    const wrapper = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
