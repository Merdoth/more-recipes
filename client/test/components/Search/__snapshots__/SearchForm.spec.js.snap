// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchForm Component snapshot it should render the right amount of elements 1`] = `
<form
  className="form-inline my-2 my-lg-0 float-menuitem-right"
>
  <input
    aria-label="Search"
    className="form-control mr-sm-2 search"
    id="search"
    name="name"
    onChange={[Function]}
    onKeyPress={[Function]}
    placeholder="Search"
    type="search"
    value=""
  />
</form>
`;
